### PR TITLE
Added NFT to README

### PR DESCRIPTION
**Pull request by Carnaux**
@kalwalt @nicolocarpignoli I believe that we can get a better gif for the NFT part, unfortunately, I don't have a better place at the moment. Any changes you suggest?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -102,6 +102,18 @@ Otherwise, as default, the script searches for places of interest near the user
...
+ **Now you can use any image you want!**
+
+
+<img height="400" width="250" src="data/images/trackingGif.gif">
+
+The software tracks interesting points in the image and from that it estimates the position of the camera. These interesting points are created using the `NFT Marker Creator`, a tool available to you for creating NFT markers, it has two versions the [WEB APP](https://carnaux.github.io/NFT-Marker-Creator/)(recommended), and the [NODE APP](https://github.com/Carnaux/NFT-Marker-Creator).
+
+If you want to understand the creation of markers in more depth, check out the  NFT Marker Creator [wiki](https://github.com/Carnaux/NFT-Marker-Creator/wiki/Creating-good-markers). 
 
 # Index
```
Comment (kalwalt) [inline README.md:118]: Maybe we can transform to **Ar.js Index** ? It seems to me like the NFT index now, but maybe it is only my impression.

**Review by kalwalt — COMMENTED** [review]
It's ok for me, What about @nicolocarpignoli ?

**Comment by kalwalt** [comment]
@Carnaux please, can you close this and re-open another PR towards https://github.com/kalwalt/AR.js/tree/nft-feature-v3 ? Last and definitive code is on that branch... Thanks!! :smile:

**Comment by kalwalt** [comment]
@Carnaux sorry if you have time make a PR instead to the **AR-js-org** to the [dev branch](https://github.com/AR-js-org/AR.js/tree) Thank you!

**Comment by Carnaux** [comment]
Sure!!

**Pull request closed without merging**